app/models/article.rb

```diff
@@ -0,0 +1,8 @@
+class Article < ApplicationRecord
+  include Visible
+
+  has_many :comments, dependent: :destroy
+
+  validates :title, presence: true
```

Comment:
When do these validations run?

Reply:
When the form of the article new or edit, is sent to the controller when the model attempt to save, check if `:title` has a value if not, returns an array of errors that will be displayed on the form.